tests/Unit/Domain/Creators/Test_Shell.cpp

```diff
@@ -22,6 +22,7 @@
 #include "Domain/CoordinateMaps/Affine.hpp"
```

Comment:
(Arbitrary line) Test the new functionality.  You don't need to cover every case, but at least one of the new versions should be checked.